pyproject.toml

```diff
@@ -0,0 +1,95 @@
+[build-system]
+requires      = ["setuptools>=61.0.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "actinia-core"
+version = "4.7.1"
+description = "Actinia Core is an open source REST API for scalable, distributed, high performance processing of geographical data that uses mainly GRASS GIS for computational tasks"
+readme = "README.md"
+authors = [
+    { name = "Anika Weinmann"},
+    { name = "Carmen Tawalika"},
+    { name = "Guido Riembauer"},
+    { name = "Jorge A. Herrera Maldonado"},
+    { name = "Julia Haas"},
+    { name = "Marc Jansen"},
+    { name = "Markus Metz"},
+    { name = "Markus Neteler"},
+    { name = "Momen Mawad"},
+    { name = "Sören Gebbert"},
+    { name = "and more"}
+]
+classifiers = [
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Programming Language :: Python :: 3",
+]
+requires-python = ">=3.8"
+keywords = ["processing", "earth observation", "grass gis"]
+dependencies = [
```

Comment:
Do we still need https://github.com/mmacata/actinia_core/blob/pypi/requirements.txt? If yes is it possible to combine it, so that we do not have to maintain the dependencies two times?

Reply:
You are right. To install actinia-core in general it is not needed anymore. But I would keep it for now until all builds are adjusted.